apps/journalfoer-soeknad/src/main/kotlin/JournalfoeringService.kt

```diff
@@ -39,7 +39,7 @@ class JournalfoeringService(private val klient: Dokarkiv) {
             journalfoerendeEnhet = finnJournalfoerendeEnhet(soeknad, gradering, forsoekFerdigstill),
             avsenderMottaker = AvsenderMottaker(id = fnrSoeker),
             bruker = Bruker(id = fnrSoeker),
-            eksternReferanseId = SOEKNAD_TITTEL + soeknadId,
+            eksternReferanseId = soeknadId,
```

Comment:
Er det potensiale for konflikter her? Eller er vi i en "adskilt" verden

Reply:
Hm, godt spørsmål. Finner ingen konkret info om det på dokarkiv sine sider

Reply:
Nei, glem det. Fant litt info: https://confluence.adeo.no/display/BOA/opprettJournalpost

Det er duplikatkontroll, så da vil ikke dette fungere. Skal endre litt